libs/feature/search/src/lib/organisations/organisations.container.component.ts

```diff
@@ -0,0 +1,53 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core'
+import { CatalogOrganisation } from '@geonetwork-ui/util/shared'
+import { Observable } from 'rxjs'
+import { filter, map } from 'rxjs/operators'
+import { SearchFacade } from '../state/search.facade'
+
+@Component({
+  selector: 'gn-ui-organisations-container',
+  templateUrl: './organisations.container.component.html',
+  styleUrls: ['./organisations.container.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrganisationsContainerComponent implements OnInit {
+  constructor(private searchfacade: SearchFacade) {}
+  organisations$: Observable<CatalogOrganisation[]>
+
+  ngOnInit(): void {
+    this.searchfacade.setConfigAggregations({
+      org: {
+        terms: {
+          size: 1000,
+          field: 'Org',
+          order: {
+            _key: 'asc',
+          },
+        },
+      },
+    })
+    this.searchfacade.requestMoreResults()
+    this.organisations$ = this.searchfacade.resultsAggregations$.pipe(
+      filter((aggs) => aggs.org),
+      map((aggs) =>
+        aggs.org.buckets
+          .map(
+            (organisation) =>
+              ({
+                name: organisation.key,
```

Comment:
you should add the count as well

Reply:
Ok, I'll add it to the model. Do we get this info from the organisation-group binding?

Reply:
Ah no I just see that there is `doc_count` in the aggregation response :-)